src/components/TableItem.vue

```diff
@@ -1,77 +0,0 @@
-<script setup lang="ts">
-import { computed, ref } from "vue"
-import { TaskInfo } from "../types";
-
-
-const emit = defineEmits<{
-  (e: "updateData", update: TaskInfo): void
-  (e: "deleteData", update: TaskInfo): void
-  (e: "addData", update: TaskInfo): void
-}>()
-
-const props = defineProps<{
-  text: string
-  complete: boolean
-  id: string
-}>()
-
-type Actions = "Edit" | "Save" | "Delete"
-
-let isEditing = ref(false)
-let textInput = ref(props.text)
-let checkBoxInput = ref(props.complete)
-
-const actionButtonText = computed<Actions>(() => {
-  if (isEditing.value) {
-    return "Save"
-  } else {
-    return "Edit"
-  }
-})
```

Comment:
Could be simplified to a ternary like
`computed<Actions>(() => isEditing.value ? 'Save' : 'Edit')`